tensorflow_lite_support/examples/task/text/desktop/python/nl_classifier_demo.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+import subprocess
+from absl import app
+from absl import flags
+import os
+
+FLAGS = flags.FLAGS
+flags.DEFINE_string('model_path', None, 'Model Path') 
+flags.DEFINE_string('text', None, 'Text to Predict') 
+
+# Required flag.
+flags.mark_flag_as_required("model_path")
+flags.mark_flag_as_required("text")
+
+def classify(model_path, text):
+    """Classifies input text into different categories
+
+    Args:
+        model_path: path to model
+        text: input text
+
+    """
+    # Run the detection tool:
+    subprocess.run(["bazel run -c opt  tensorflow_lite_support/examples/task/text/desktop:nl_classifier_demo --  --model_path=" + model_path + "  --text='" + text + "'"], shell=True)
```

Comment:
Just realized that bazel run won't work if we want to distribute the demo to the Support Pypi package. We should include a prebuilt C++ demo binary.

@xunkai55 how does it work to include prebuilt C++ binaries when building the Support Pypi package. If we let this python demo depend on the C++ demo cc_binary, will it work?

Reply:
Firstly, will this example (and the whole `examples/task/**/*.py`) plan to be packaged into pip package? If not, it's fine to put anything here.

If we do want to distribute the examples in the pip package (or a new standalone pip package), I would recommend using `pybind11` to call the C++ library rather than using subprocess to call an arbitrary binary (Task library Python API). It's much cleaner, and already proved working in our infrastructure.

Reply:
Thanks Xunkai for the input! The main purpose of developing this Python demo is to distribute it in the pip package, and use it as a workaround before we have the formal Python API. 

Using `pybind11` to call C++ requires a refactoring of the C++ demo, and will increase the workload for our external  contributor a lot. Let's first check in this PR. I'll change bazel run to the binary internally, and try figuring out if it's possible to add it to Pypi. If it fails, we can then update the underlying logic to using pybind.